packages/block-library/src/page-list/convert-to-links-modal.js

```diff
@@ -13,12 +13,12 @@ export function ConvertToLinksModal( { onClick, onClose, disabled } ) {
 		<Modal
 			onRequestClose={ onClose }
 			title={ __( 'Edit Page List' ) }
-			className={ 'wp-block-page-list-modal' }
+			className="wp-block-page-list-modal"
 			aria={ {
 				describedby: 'wp-block-page-list-modal__description',
 			} }
 		>
-			<p id={ 'wp-block-page-list-modal__description' }>
+			<p id="wp-block-page-list-modal__description">
```

Comment:
Should be quite straightforward to use `useInstanceId()` or something similar here, no? Otherwise, this will create a `no-restricted-syntax` that wasn't reported previously.